biome.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I guess it has been generated using an init command or something similar, most likely the majority of those settings are defaults.
I would keep it clean, including only options that differs from defaults (eventually remove the file entirely for now).

Reply:
I thought I added this info somewhere but apparently not. Basically it's not simply been run with an init command but I've ported the previously prettier configuration with a specific biome command which is: biome migrate prettier --write

I've done it to reduce the number of changes when applying the biome fixes.

The default one is empty and works as well. Your choice

Reply:
So if the settings have been migrated there shouldn't be format changes, but there are. 
Maybe biome and prettier are not entirely compatible or the migrate feature has some caveats.

Btw keep it as it is, I'll get rid of redundant stuff later.

Reply:
You're starting from the failed assumption that biome is a replacement for prettier but it's actually a replacement for eslint as well, that's the reason why you see changes. I don't want to explain what biome is or how it works, I'm sure you know there is a documentation you can check.